jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/APIKeys/ConfigureApiRestriction.java

```diff
@@ -87,7 +87,10 @@ private boolean checkForRestrictAPI(String vmEsURL, Map<String, Object> mustFilt
             logger.info("hit array size {}",hitsJsonArray.size());
             JsonObject apiKeys = (JsonObject) ((JsonObject) hitsJsonArray.get(0))
                     .get(PacmanRuleConstants.SOURCE);
-
+            //handling "restrictions" attribute being NULL
+         
+            if(apiKeys.get(PacmanRuleConstants.RESTRICTIONS).isJsonNull())
+                return true;
             if (apiKeys.get(PacmanRuleConstants.RESTRICTIONS).getAsJsonObject()!=null &&apiKeys.get(PacmanRuleConstants.RESTRICTIONS).getAsJsonObject().size()>0) {
```

Comment:
1. always use braces
2. are sure about returning `true`?
3. how you can apply the same approach of setting `validationResult` to some value instead of returning right here? Return here brings inconsistency - whole method except this part uses another approach to return value.